gnovm/cmd/gno/test.go

```diff
@@ -267,14 +268,15 @@ func gnoTestPkg(
 
 	// testing with *_test.gno
 	if len(unittestFiles) > 0 {
-		memPkg := gno.ReadMemPackage(pkgPath, pkgPath)
+		pkgName := strings.TrimPrefix(pkgPath, "./examples/")
```

Comment:
This is were the test package is changed, the `./examples/` prefix is removed. 

Maybe we should introduce a constant.